Spotishell/Public/Backup-Library.ps1

```diff
@@ -0,0 +1,80 @@
+<#
+    .SYNOPSIS
+        Backup Library items to json file (FollowedArtists, Playlists, SavedAlbums, SavedShows, SavedTracks)
+    .EXAMPLE
+        PS C:\> Backup-Library -Path '.\mySpotifyBackup.json'
+        Backup all Library items into '.\mySpotifyBackup.json'
+    .EXAMPLE
+        PS C:\> Backup-Library -Type FollowedArtists,SavedAlbums -Path '.\mySpotifyBackup.json'
+        Backup Followed Artists and Saved Albums items into '.\mySpotifyBackup.json'
+    .PARAMETER Path
+        Path of the backup file you want to create
+    .PARAMETER Type
+        One or more items type to backup (All, FollowedArtists, SavedAlbums, SavedShows, SavedTracks)
+    .PARAMETER ApplicationName
+        Specifies the Spotify Application Name (otherwise default is used)
+#>
+function Backup-Library {
+    param (
+        [Parameter(Mandatory)]
+        [string]
```

Comment:
```suggestion
        [ValidateScript({
            if( -Not ($_ | Test-Path) ){
                throw "File or folder does not exist"
            }
            return $true
        })]
```

Reply:
this cmdlet create the backup, so there is no problem if the file doesn't exists.